packages/admin/README.md

```diff
@@ -40,6 +40,43 @@ await client.createProposal({
 });
 ```
 
+You can also optionally set the `simulate` flag as part of the `createProposal` request (as long as this is not a batch proposal) to simulate the proposal within the same request. You can override simulation parameters by setting the `overrideSimulationOpts` property, which is a `SimulationRequest` object.
```

Comment:
For my own knowledge. Does it mean adding simulate will not actually create the proposal? What will be simulated the proposal executed if it gets executed?

Reply:
So the proposal will be created, but the added benefit is that we also trigger a simulation for that proposal within the same request. So it can be done as part of 1 call, rather than 2 separate calls for the user (createProposal + simulateProposal).

Reply:
oh I see thanks for clarification (nit could be in MR desciption to give context is original MR on defender was not reviewd maybe, or link other MR 🤔)